### ISSUE TITLE
Add link to source code in README

### ISSUE THREAD

**Issue by bittner** [peter-evans/dockerhub-description#16]
It's difficult to find this source code repository if you only know the [Docker Hub location](https://hub.docker.com/r/peterevans/dockerhub-description) of the created image.

Could you add a link back to the source code repository, and maybe make the link to the LICENSE file an absolute URL?

**Comment by peter-evans** [comment]
Hi @bittner 

You're right, it's not easily linked to. I've made the main readme title into a link to this repository, and updated the license link to an absolute URL.

Thanks!

**Comment by gerlero** [comment]
Unfortunately, this doesn't seem to work anymore (if I understand it correctly, this would make the main heading on Docker Hub link to the source GitHub repo, correct?)

Do you have any other suggestion or workaround for getting a link back to GitHub automatically on the Docker Hub description? I know I can manually make the main heading of my README link to the GitHub repo, but that would be somewhat confusing for users seeing it on GitHub...

**Comment by peter-evans** [comment]
Hi @gerlero 

It does still work, I just forgot to execute the workflow that updates the links after I did some testing a few months ago. I've run [this workflow](https://github.com/peter-evans/dockerhub-description/blob/main/.github/workflows/dockerhub-description.yml) again and it's updated the links in the readme [here](https://hub.docker.com/repository/docker/peterevans/dockerhub-description).

**Comment by gerlero** [comment]
@peter-evans thanks!

I had it wrong and thought this was performed by the action itself and not as a separate step. I've now added an extra step similar to the one in the workflow you linked.

FWIW, I came up with a generic substitution for the top-level heading with:
```sh
sed -i "s|^\s*#\s\+\(.\+\)|# [\1]($GITHUB_SERVER_URL/$GITHUB_REPOSITORY)|" README.md
```
which should do the trick for many repos without changes.

Thanks for making this useful action!
